lib/actions/logs.coffee

```diff
@@ -42,14 +35,13 @@ module.exports =
 	]
 	permission: 'user'
 	action: (params, options, done) ->
-		resin.logs.subscribe params.uuid, {
+		resin.logs.subscribe(params.uuid, {
 			history: options.num or LOGS_HISTORY_COUNT
 			tail: options.tail
-		}, (error, message) ->
-			return done(error) if error?
+		}, (message) ->
 			if _.isArray(message)
 				_.each message, (line) ->
 					console.log(line.message)
 			else
 				console.log(message.message)
-			return done()
+	  ).nodeify(done)
```

Comment:
This should be indented one more level, to be aligned with `resin.logs.subscribe`.